integration/api/src/main/java/com/cerner/beadledom/integration/api/model/OffsetPaginatedListHelloWorldDto.java

```diff
@@ -0,0 +1,49 @@
+package com.cerner.beadledom.integration.api.model;
+
+import com.cerner.beadledom.pagination.models.OffsetPaginatedListDto;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.google.auto.value.AutoValue;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Client implementation of an offset based pagination list of {@link HelloWorldDto}.
+ *
+ * @author Nick Behrens
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@AutoValue
+@JsonDeserialize(builder = OffsetPaginatedListHelloWorldDto.Builder.class)
+public abstract class OffsetPaginatedListHelloWorldDto implements OffsetPaginatedListDto<HelloWorldDto> {
+
+  @JsonProperty("items")
+  public abstract List<HelloWorldDto> items();
```

Comment:
Is this needed? Does it not come from the interface automatically?

Reply:
It isn't needed, good catch: https://github.com/cerner/beadledom/pull/149/commits/2c3d8ca7133d1342f729c18d5e2ef5d56ae33db5